src/Files.App/ViewModels/SettingsViewModels/AppearanceViewModel.cs

```diff
@@ -224,7 +224,51 @@ public bool ShowFileTagsSection
             }
         }
 
-        public async Task OpenThemesFolder()
+		#region WidgetsPage Widgets
+
+		public bool ShowFoldersWidget
+		{
+			get => UserSettingsService.WidgetsSettingsService.ShowFoldersWidget;
+			set
+			{
+				if (value != UserSettingsService.WidgetsSettingsService.ShowFoldersWidget)
+					UserSettingsService.WidgetsSettingsService.ShowFoldersWidget = value;
+			}
+		}
+
+		public bool ShowDrivesWidget
+		{
+			get => UserSettingsService.WidgetsSettingsService.ShowDrivesWidget;
+			set
+			{
+				if (value != UserSettingsService.WidgetsSettingsService.ShowDrivesWidget)
+					UserSettingsService.WidgetsSettingsService.ShowDrivesWidget = value;
+			}
+		}
+
+		public bool ShowBundlesWidget
+		{
+			get => UserSettingsService.WidgetsSettingsService.ShowBundlesWidget;
+			set
+			{
+				if (value != UserSettingsService.WidgetsSettingsService.ShowBundlesWidget)
+					UserSettingsService.WidgetsSettingsService.ShowBundlesWidget = value;
+			}
+		}
+
+		public bool ShowRecentFilesWidget
+		{
+			get => UserSettingsService.WidgetsSettingsService.ShowRecentFilesWidget;
+			set
+			{
+				if (value != UserSettingsService.WidgetsSettingsService.ShowRecentFilesWidget)
```

Comment:
Settings should be moved to AppearanceSettingsService

Reply:
Should Widget-Settings be logged?

Reply:
Yes